README.md

```diff
@@ -90,6 +90,34 @@ Before merging the latest changes from `develop` into a release branch:
 * All build artifacts of a library project should be in the directory `./lib`
 * All documentation should be below the directory `./docs`
 
+### Logging 
+
+* [Winston](https://github.com/winstonjs/winston) should be used as logging library.
+* By default logging should be enabled on the console.
```

Comment:
`By default logging should be enabled on the console` ==> `By default logging should be enabled to the console`